### ISSUE TITLE
Feature request: button for adding files and folders

### ISSUE THREAD

**Issue by rhz** [DeaDBeeF-Player/deadbeef#1915]
I would be nice to have a button to add files and folders (perhaps to the left of the stop button).

When I want to add a file to the playlist, sometimes I'm distracted and press "open file" instead of "add file" in the file menu, which removes everything else from the playlist. By having a button, I'd not need to go to the file menu, where the dangerous "open file" is. If I want to remove a song or a set of songs from the playlist, I can do it with the delete key.

Thank you for the nice program, it's been my favourite music player for a while now.

**Comment by Alexey-Yakovenko** [comment]
It is possible to customize the UI, and add a button on your own.
Enter design mode, right click on the button bar, change the number of items in it.
Then right click on the new slot, and add a button. Then right click on the button, and assign the add files/folders button to it.

You probably want to backup your config file (~/.config/deadbeef/config) in case you mess it up by accident.

**Comment by rhz** [comment]
Thanks! That's cool.

I'm not sure how to change the number of items in the button bar. When right-clicking on the button bar, the only options I get are "replace with...", "delete", "cut", "copy", "paste", "expand", and "fill".

**Comment by Alexey-Yakovenko** [comment]
ok.. I was a little wrong.. been a while since I did that last time.

You can't modify the playbox, but you can still make a button..

You can replace the seekbar with HBox, make HBox contain 2 items, and uncheck the "Homogeneous" checkbox.

Then insert a button on the left, and re-insert seekbar on the right.

Then uncheck expand/fill checkboxes on the button.

**Comment by rhz** [comment]
Right. I replaced the playback controls with an hbox instead and followed your instructions. It works but unfortunately there's a lot of empty space now after the playback controls and before the seekbar. I unchecked the homogeneous checkbox and tried will all possible combinations of fill and expand for the hbox, button and playback controls.

I also tried editing in designer mode the file menu to remove the "open file" entry but it's not possible.
